AEM/getting-started-with-automixedprecisionmkl.md

```diff
@@ -199,15 +161,27 @@ python conv2D_bf16.py
 2022-08-19 00:06:08.750799: I tensorflow/core/grappler/optimizers/auto_mixed_precision.cc:2303] Running auto_mixed_precision_onednn_bfloat16 graph optimizer
 2022-08-19 00:06:08.751058: I tensorflow/core/grappler/optimizers/auto_mixed_precision.cc:2240] Converted 3/11 nodes to bfloat16 precision using 0 cast(s) to bfloat16 (excluding Const and Variable casts)
 ```
-The data-flow graph after porting the model to BFloat16 type looks like below.
+> _No allowlist ops found typically refers to the data session._
+> One can also use ONEDNN_VERBOSE logs as discussed earlier to verify AMX BF16 instructions.
 
-![amp-mkl2](/content/dam/develop/external/us/en/images/automixedprecisionmkl-2.png)
+## IV. Tips for best performance on Intel® Xeon CPUs
```

Comment:
This seems unelated to mixed precision, can we remove this section?

Reply:
I added this to section to just bring awareness. But yes, I will remove it.

Reply:
You can link to any existing optimization document.